### DISCUSSION TITLE
Friendly competitor invitation

### DISCUSSION THREAD

**Discussion by WebDB-App** [dbeaver/dbeaver#22526] [General]
Hello Guys,

First, thanks you for everything you did to the open source community
I just want to invite people to take a look at a new competitor, if possible give feedbacks and even to participate -> webdb.app

Have a great day and happy new year =)
I sincerly hope you doesn't consider this as a awkward move, I just want to help open source community

**Comment by E1izabeth** [comment]
Hello. We wish success to your product =)

**Reply by WebDB-App** [reply]
Hey @E1izabeth thanks for taking the invitation friendly

I have such respect your product and community

If you have time to give feedback, It will be lovely
If you want my opinion on your, I can give you but I don't think you need some at your point

**Reply by E1izabeth** [reply]
Probably, you have a typo here on the web site. I suppose you mean AI assistant
![image](https://github.com/dbeaver/dbeaver/assets/28875055/6bad3a0e-7708-443e-97f1-2c56755c16d5)

**Comment by WebDB-App** [comment]
Hey, thanks, it's fixed

Le lun. 22 janv. 2024 à 10:32, Elizabeth ***@***.***> a
écrit :
